### PR TITLE
Update payments-api-overview per IA Refactor

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-2065](https://jira.bigcommerce.com/browse/DEVDOCS-2065)

## What changed?
Update per IA Refactor guidelines

### CODE REVIEW

File: docs/api-docs/payments/payments-api-overview.md [hunk 1]
```diff
@@ -2,120 +2,121 @@
...
+BigCommerce is only responsible for the security of credit cards to the extent that secure handling is maintained while the payment is en route from payment request to payment processors. As a third-party developer, you are responsible for developing the storefronts or recurring billing apps in a PCI compliant manner. You will also need to maintain a PCI compliance certification for third-party service providers certified by an external Qualified Security Assessor (QSA). 
+
+Merchants or shoppers' personal identifiable information (PII) collected by recurring billing apps that consumes the BigCommerce Payments API must have its own Privacy Policy sufficient to the requirements of the European Union General Data Protection Requirements (GDPR). The GDPR must be available and displayed to the general public.
```
Comment (ektravel) [inline docs/api-docs/payments/payments-api-overview.md:35]:
```suggestion
Merchants or shoppers' personal identifiable information (PII) collected by recurring billing apps that consume the BigCommerce Payments API must have its own Privacy Policy sufficient to the requirements of the European Union General Data Protection Requirements (GDPR). The GDPR must be available and displayed to the general public.
```

File: docs/api-docs/payments/payments-api-overview.md [hunk 2]
```diff
@@ -2,120 +2,121 @@
...
-**The following gateways are supported for stored cards:**
+**BigCommerce supports the following gateways for stored cards:**
 
 * AdyenV2
 * Authorize.net
 * CyberSource
 * MyVirtualMerchant
 * Paymetric
-* Paypal Powered by Braintree
+* Paypal powered by Braintree
```
Comment (ektravel) [inline docs/api-docs/payments/payments-api-overview.md:62]:
```suggestion
* PayPal powered by Braintree
```

File: docs/api-docs/payments/payments-api-overview.md [hunk 3]
```diff
@@ -300,25 +301,25 @@ lineNumbers: true
...
+## Credit cards
 
 There are two steps to using a credit card to make a payment.
 
 1. [Create Access Token](/api-reference/payments/payments-create-payment-token-api/payment-access-token/paymentsaccesstokenspost)
 2. [Process Payment](/api-reference/payments/payments-process-payments/payment/paymentspost)
 
-### Create Access Token
-1. Make a request to [Create Access Token](/api-reference/payments/payments-create-payment-token-api/payment-access-token/paymentsaccesstokenspost) to to get the authorization token that needs to be passed in the header when processing the payment. The ID of the order needs to be part of the request body.
+### Create access Token
```
Comment (ektravel) [inline docs/api-docs/payments/payments-api-overview.md:313]:
```suggestion
### Create access token
```

File: docs/api-docs/payments/payments-api-overview.md [hunk 4]
```diff
@@ -346,11 +347,11 @@ lineNumbers: true
 }
 ```
 
-### Process the Payment
+### Process the payment
 
 2. To process the payment, send a POST to [Process Payment](/api-reference/payments/payments-process-payments/payment/paymentspost).
```
Comment (ektravel) [inline docs/api-docs/payments/payments-api-overview.md:352]:
```suggestion
2. To process the payment, send a `POST` request to [Process Payment](/api-reference/payments/payments-process-payments/payment/paymentspost).
```

File: docs/api-docs/payments/payments-api-overview.md [hunk 5]
```diff
@@ -526,69 +527,69 @@ lineNumbers: true
...
+The following is a list of supported gateways and a list of their test credit cards. These can be useful while getting your app setup. Check your credit card setup in both [BigCommerce](https://support.bigcommerce.com/s/article/Online-Payment-Methods#setup) and the payment gateway to make sure they are configured properly. If the credit cards do not work or stop working, please reach out to the payment provider as these are not maintained by BigCommerce.
 
 * [Authorize.Net](https://developer.authorize.net/hello_world/testing_guide/)
-* [Paypal Powered by Braintree](https://developers.braintreepayments.com/guides/credit-cards/testing-go-live/php)
+* [Paypal powered by Braintree](https://developers.braintreepayments.com/guides/credit-cards/testing-go-live/php)
```
Comment (ektravel) [inline docs/api-docs/payments/payments-api-overview.md:537]:
```suggestion
* [PayPal powered by Braintree](https://developers.braintreepayments.com/guides/credit-cards/testing-go-live/php)
```

File: docs/api-docs/payments/payments-api-overview.md [hunk 6]
```diff
@@ -526,69 +527,69 @@ lineNumbers: true
...
 |`30100` | Payment access token could not be created. | N/A|N/A|
-| `30101` | Order is invalid. | The order is in the wrong status. | Orders must be in Incomplete Status with a `status_id:0` <br>  The order must be created by the Checkout SDK, Checkout API or V2 Orders API. Orders created in the Control and set to an incomplete status will return this error. |
-| `30102` | The payment was declined. | The card information provided was incorrect<br>The token provided was incorrect | Check that the provider shopper information is correct<br>Make sure the token in the Authorization header field is correct |
-| `30103` | Card has expired |N/A | N/A|
+| `30101` | Order is invalid. | The order is in the wrong status. | Orders must be in Incomplete Status with a `status_id:0` <br>  The order must be created by the Checkout SDK, Checkout API or V2 Orders API. Orders created in the control panel and set to an incomplete status will return this error. |
```
Comment (ektravel) [inline docs/api-docs/payments/payments-api-overview.md:590]:
```suggestion
| `30101` | Order is invalid. | The order is in the wrong status. | Orders must be in Incomplete Status with a `status_id:0`. <br>  The order must be created by the Checkout SDK, Checkout API, or V2 Orders API. Orders created in the control panel and set to an incomplete status will return this error. |
```

File: docs/api-docs/payments/payments-api-overview.md [hunk 7]
```diff
@@ -526,69 +527,69 @@ lineNumbers: true
...
+| `30102` | The payment was declined. | The card information provided was incorrect<br>The token provided was incorrect | Check that the shopper information provided is correct<br>Make sure the token in the authorization header field is correct |
```
Comment (ektravel) [inline docs/api-docs/payments/payments-api-overview.md:591]:
```suggestion
| `30102` | The payment was declined. | The card information provided was incorrect.<br>The token provided was incorrect. | Check that the shopper information provided is correct.<br>Make sure the token in the authorization header field is correct. |
```

File: docs/api-docs/payments/payments-api-overview.md [hunk 8]
```diff
@@ -2,120 +2,121 @@
...
 - [Using the Orders API](#using-the-orders-api)
-- [Technical Details](#technical-details)
-- [Sample App Diagram](#sample-app-diagram)
-- [Error Codes](#error-codes)
+- [Technical details](#technical-details)
+- [Sample app diagram](#sample-app-diagram)
+- [Error codes](#error-codes)
 - [FAQ](#faq)
-- [Resources](#resources)
+- [Additional resources](#resources)
```
Comment (ektravel) [inline docs/api-docs/payments/payments-api-overview.md:15]: We can keep Resources as a header and add Additional resources as a sub-header. 
```suggestion
- [Resources](#resources)
```

**Review by uwikaiddi — APPROVED** [review]

**Review by uwikaiddi — APPROVED** [review]

**Pull request merged**